packages/core/src/State.ts

```diff
@@ -215,7 +215,7 @@ export class State<
    * @param context
    */
   public static inert<TC, TE extends EventObject = EventObject>(
-    stateValue: State<TC, TE> | StateValue,
+    stateValue: State<TC, TE, any, any, any> | StateValue,
```

Comment:
Good candidate for `AnyState`

Reply:
Not really though, cause we'd lose the ability to pass `TC` and `TE` here. Unless maybe something like this would be implemented:
```diff
-type AnyState = State<any, any, any, any, any>
+type AnyState<
+  TContext = any,
+  TEvent extends EventObject = any,
+  TStateSchema extends StateSchema<TContext> = any,
+  TTypestate extends Typestate<TContext> = any,
+  TResolvedTypesMeta = any
+> = State<TContext, TEvent, TStateSchema, TTypestate, TResolvedTypesMeta>;
```

Although I would probably try to stay away from changing this across the codebase etc. This might become a refactor target for v5 branch but given that merging main to next will already be a nightmare I feel that it's better to hold off doing that for now